app/code/Magento/CurrencySymbol/Block/Adminhtml/System/Currency.php

```diff
@@ -41,7 +41,14 @@ protected function _prepareLayout()
             ]
         );
 
-        $onClick = "setLocation('" . $this->getUrl('adminhtml/system_config/edit/section/currency') . "')";
+        $currencyOptionPath = $this->getUrl(
+            'adminhtml/system_config/edit',
+            [
+                'section' => 'currency',
+                '_fragment' => 'currency_options-link'
+            ]
+        );
+        $onClick = "setLocation('" . $currencyOptionPath . "')";
```

Comment:
You may simplify this statement slightly

```suggestion
        $onClick = "setLocation('$currencyOptionPath')";
```

Reply:
Thank you for your suggestion :)